windowsservercore-1709/Dockerfile

```diff
@@ -1,9 +1,10 @@
-FROM microsoft/windowsservercore:1709
+# escape=`
+FROM microsoft/windowsservercore:ltsc2016
 
-RUN powershell -NoProfile -Command \ 
-    Add-WindowsFeature Web-Server \
-    Invoke-WebRequest https://dotnetbinaries.blob.core.windows.net/servicemonitor/2.0.1.1/ServiceMonitor.exe -OutFile C:\ServiceMonitor.exe
+RUN powershell -Command `
+    Add-WindowsFeature Web-Server; `
+    Invoke-WebRequest -UseBasicParsing -Uri "https://dotnetbinaries.blob.core.windows.net/servicemonitor/2.0.1.1/ServiceMonitor.exe" -OutFile "C:\ServiceMonitor.exe"
 
 EXPOSE 80
 
-ENTRYPOINT ["C:\\ServiceMonitor.exe", "w3svc"]
+ENTRYPOINT ["C:\\ServiceMonitor.exe", "w3svc"]
```

Comment:
Nit: newline

Reply:
Why is the windowsservercore-1709 Dockerfile now coming FROM the ltsc2016 base image? Isn't that going to double the image size?

Reply:
@IanOfOz Thanks for catching that. It was an accident that has been fixed in 619c3d052d3828e25f318e740fb051c5295f1e8c